infrastructure/tooling/src/smoketest/checks/builtin.js

```diff
@@ -1,5 +1,13 @@
 const taskcluster = require('taskcluster-client');
 
+exports.scopeExpression = {
```

Comment:
We'll have to be careful to keep these synchronized with whatever the actual scopes are for various smoketests right? This feels like it will go out of date quickly iiuc

Reply:
Yeah, the best I could do was to make it per-file (so this chunk is just the scopes for the built-in-workers smoketests).